tests/cases/realworld/tslint/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "tslint-test",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "author": "",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "tslint": "^5.8.0",
+    "typescript": "^2.5.3"
+  },
+  "devDependencies": {
```

Comment:
should we also generate package.json in order to be able to copy `devDependencies` from the newly installed `tslint`? (Or just change these to latest, as you suggested in person :)

Reply:
Changed to `latest`.